ci/jenkins/test-vm.sh

```diff
@@ -143,10 +143,13 @@ function apply_antrea {
     fi
     TEMP_ANTREA_TAR="antrea-image.tar"
     docker save antrea/antrea-agent-ubuntu:latest antrea/antrea-controller-ubuntu:latest -o $TEMP_ANTREA_TAR
-    ctr -n k8s.io image import $TEMP_ANTREA_TAR
+    IP=$(kubectl get nodes --selector=node-role.kubernetes.io/control-plane= --no-headers=true -o custom-columns=IP:.status.addresses[0].address)
+    rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" $TEMP_ANTREA_TAR jenkins@${IP}:${WORKDIR}/$TEMP_ANTREA_TAR
+    ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "crictl rmi --prune; crictl ps --state Exited; ctr -n=k8s.io images import ${WORKDIR}/$TEMP_ANTREA_TAR" || true
```

Comment:
So we only copy / import the image on the control plane Node, which I think is consistent with what we were doing before.
But in this case, how does the image become available to the other (worker) Nodes? Is there a different step where worker Nodes are added and we load the required images?

Reply:
@antoninbas The vm-e2e testbed includes only one controller node in the kubernetes cluster, other nodes are non-Kubernetes rather than Kubernetes worker nodes(Their image is copied and installed using `copy_antrea_agent_files_on_linux/windows`, which differs from the one used in the Kubernetes cluster). 

Previously, we built Antrea image directly on the controller node, so there was no need to copy or import it again. However, we now need to build the image on the jumper node and then deliver it to the k8s controller node.

Reply:
Thanks for the explanation, I didn't realize we didn't have any separate K8s worker Nodes for this test

Reply:
(BTW, it seems that we could still have iterated over all Nodes to load the images, not just the ones with the `node-role.kubernetes.io/control-plane` label, to be on the safe side, but the current version is fine if we don't expect to add regular K8s worker Nodes for this test in the future).

Reply:
> (BTW, it seems that we could still have iterated over all Nodes to load the images, not just the ones with the `node-role.kubernetes.io/control-plane` label, to be on the safe side, but the current version is fine if we don't expect to add regular K8s worker Nodes for this test in the future).

Yes, this will be better for future maintenance. I've updated the related code, thanks for the suggestion.